src/components/utils/popover/popover.types.ts

```diff
@@ -64,7 +92,23 @@ export interface PopoverEventMap {
   /**
    * Fired when popover closes
    */
-  [PopoverEvent.Close]: undefined;
+  [PopoverEvent.Closed]: undefined;
+
+  /**
+   * Fired when popover closes because item with 'activateOnClose' property set was clicked
+   * Value is the item that was clicked
+   */
+  [PopoverEvent.ClosedOnActivate]: undefined;
+
+  /**
+   * Fired when nested popover opens
+   */
+  [PopoverEvent.OpenedNestedPopover]: OpenedNestedPopoverEventPayload;
+
+  /**
+   * Fired when nested popover closes
+   */
+  [PopoverEvent.ClosedNestedPopover]: undefined;
```

Comment:
```suggestion
  [PopoverEvent.NestedPopoverClosed]: undefined;
```

Reply:
these events are not needed anymore, removed them